skymp5-front/src/constructor.js

```diff
@@ -90,11 +90,12 @@ const Constructor = props => {
 
       const [hints, setHints] = useState(hintsarr);
       let hintIndex = 0;
-      bodylines.forEach((line) => {
+      bodylines.forEach((line, lineIndex) => {
         let arr = [];
-        line.forEach((obj) => {
+        line.forEach((obj, elementIndex) => {
           let curElem = undefined;
           let hasHint = obj.element.hint != undefined ? true : false;
+          let key = lineIndex + elementIndex + obj.element.type;
```

Comment:
There will be collisions. Maybe something like lineIndex + "_" + elementIndex + "_" + obj.element.type?